plugins/TimingTriggerCandidateMaker.cpp

```diff
@@ -57,38 +57,33 @@ void
 TimingTriggerCandidateMaker::do_conf(const nlohmann::json& config)
 {
   auto params = config.get<dunedaq::trigger::timingtriggercandidatemaker::Conf>();
-  //			try {
-  m_map.push_back({ params.s0.time_before, params.s0.time_after });
-  m_map.push_back({ params.s1.time_before, params.s1.time_after });
-  m_map.push_back({ params.s2.time_before, params.s2.time_after });
-  //			} catch(...)  {
-  //				ERS_LOG(get_name() << " unsuccessfully configured");
-  //			}
-  //			ERS_LOG(get_name() << " successfully configured");
+  try {
+    m_detid_offsets_map.push_back({ params.s0.time_before, params.s0.time_after });
+    m_detid_offsets_map.push_back({ params.s1.time_before, params.s1.time_after });
+    m_detid_offsets_map.push_back({ params.s2.time_before, params.s2.time_after });
+  } catch (const ers::Issue& excpt) {
+    throw dunedaq::trigger::ConfigurationError(ERS_HERE, get_name(), "unsuccessfully configured", excpt);
+  }
 }
 
 void
 TimingTriggerCandidateMaker::init(const nlohmann::json& iniobj)
 {
-  auto qi = appfwk::queue_index(iniobj, { "input", "output" });
-  // try {
-  inputQueue_.reset(new source_t(qi["input"].inst));
-  //} catch (const ers::Issue& excpt) {
-  //	throw dunedaq::dunetrigger::InvalidQueueFatalError(ERS_HERE, get_name(), "input", excpt);
-  //}
-
-  // try {
-  outputQueue_.reset(new sink_t(qi["output"].inst));
-  //} catch (const ers::Issue& excpt) {
-  //	throw dunedaq::dunetrigger::InvalidQueueFatalError(ERS_HERE, get_name(), "output", excpt);
-  //}
+  try {
+    auto qi = appfwk::queue_index(iniobj, { "input", "output" });
+    inputQueue_.reset(new source_t(qi["input"].inst));
+    outputQueue_.reset(new sink_t(qi["output"].inst));
+  } catch (const ers::Issue& excpt) {
+    throw dunedaq::trigger::InvalidQueueFatalError(ERS_HERE, get_name(), "input/output", excpt);
+  }
 }
 
 void
 TimingTriggerCandidateMaker::do_start(const nlohmann::json&)
 {
   thread_.start_working_thread();
-  // ERS_LOG(get_name() << " successfully started");
+  std::string oss_prog = get_name() + " successfully started";
+  ers::debug(dunedaq::trigger::ProgressUpdate(ERS_HERE, get_name(), oss_prog));
```

Comment:
I think `ers::debug(issue)` should become `TLOG_DEBUG(level) << issue;` where `level` is 0, 1, 2, .... Higher levels are more verbose.